crates/bevy_ui/src/render/mod.rs

```diff
@@ -243,15 +243,11 @@ pub fn extract_default_ui_camera_view<T: Component>(
             camera.physical_viewport_rect(),
             camera.physical_viewport_size(),
         ) {
-            let mut projection = OrthographicProjection {
-                far: UI_CAMERA_FAR,
-                window_origin: WindowOrigin::BottomLeft,
-                ..Default::default()
-            };
-            projection.update(logical_size.x, logical_size.y);
+            let projection_matrix =
+                Mat4::orthographic_rh(0.0, logical_size.x, logical_size.y, 0.0, 0.0, UI_CAMERA_FAR);
```

Comment:
If this is where the actual "inversion" happens, I'd suggest to add a short comment here for future reference

Reply:
I added a comment. Do you think it's clear enough?

Reply:
That should work, thanks!